tests/integ/test_stored_procedure.py

```diff
@@ -433,9 +433,6 @@ def plus4_then_mod5(session_, x):
             return mod5(session_, session_.sql(f"SELECT {x} + 4").collect()[0][0])
 
         stage_file = f"@{tmp_stage_name}/{os.path.basename(test_files.test_sp_py_file)}"
-        Utils.upload_to_stage(
-            session, tmp_stage_name, test_files.test_sp_py_file, compress=False
-        )
```

Comment:
this code has no effect on test running?

Reply:
apparently not. There is a similar test in `test_udf.py` which gave me a clue. merge gates are happy as you can see.

Reply:
Uploading **ONE** existing file no longer works in Local FS because it fails at the step with `ls <path_to_single_fail>`  in stored proc connector's file transfer agent code. This command will only fail (1) in Local FS and (2) when the file already exists. So fixing the test by removing an additional upload makes sense to me.

Reply:
thanks for the explanation